### PR TITLE
S3 to sqs

### PR DESCRIPTION
**Pull request by rajrkane**
Receives objects from S3 output bucket, parses them, and sends messages to SQS response queue.

Error handling can be added in a new branch.

Test: `python app-tier/app.py`

### CODE REVIEW

File: app-tier/init.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+bucket = S3() 
+response_queue = SQS()
+
+# for each result, send message to queue
+for res in bucket.results['results']:
+    msg = {
+        'name': res['key'],
+        'body': res['body']
+    }
+    response_queue.send_message(msg)
```
Comment (tmanuszak) [inline app-tier/init.py]: Is this just practice code to see if the app's interaction with S3 and SQS works? If so, this works.

The real functionality of the app should be
- pull a message from the requests
- pull the corresponding image from input bucket
- classify the image
- put the classification in the output bucket
- send a message to the response queue.

Correct?

Comment (rajrkane) [reply app-tier/init.py]: This covers sending a message from a worker to the response queue (last bullet point). The classifier and workers were not available prior to this,  so the first points are not covered here.

This also pulls a classification that has already been placed in the output bucket. I suppose the worker can just return the classification without pulling from S3, making that part unnecessary.

Comment (rajrkane) [reply app-tier/init.py]: Let me push changes to include the penultimate point: putting the classification in the output bucket. The last two points should suffice for this PR.

**Comment by tmanuszak** [comment]
Okay makes sense.

File: app-tier/init.py (hunk 1 above)
Comment (tmanuszak) [reply app-tier/init.py]: Okay makes sense.

**Pull request merged**